<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>1.Stereo</string>
    </property>
    <widget class="QPushButton" name="Button_1_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1.1 Disparity</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>221</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>2.Background subtraction</string>
    </property>
    <widget class="QPushButton" name="Button_2_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 Background subtraction</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>3.Feature Tracking</string>
    </property>
    <widget class="QPushButton" name="Button_3_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>3.1 Preprocessing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_3_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>3.2 Video Tracking</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>4. Augmented Reality</string>
    </property>
    <widget class="QPushButton" name="Button_4_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>4.1 Augumented Reality</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>637</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
